### DISCUSSION TITLE
Help: skip checkup error-  Xcode version: error executing xcodebuild

### DISCUSSION THREAD

**Discussion by Harryuxd** [skiptools/skip#3] [Q&A]
<img width="583" alt="Screenshot 2023-10-26 at 8 47 18 PM" src="https://github.com/skiptools/skip/assets/47468239/24c9c8b0-6302-4cb6-859e-fec79e844ad6">

Xcode is installed, not sure what is going on

**Comment by marcprux** [comment]
Can you run `xcode-select --install` and let us know the output?

And then run `skip doctor --verbose` and let us know if it still fails?

**Comment by Harryuxd** [comment]
Resolved!!!
Thanks
